runtime/compiler/optimizer/EscapeAnalysis.cpp

```diff
@@ -9691,11 +9691,8 @@ bool TR_LocalFlushElimination::examineNode(TR::Node *node, TR::TreeTop *tt, TR::
          if (opCode.hasSymbolReference())
             {
             TR::SymbolReference *symReference = node->getSymbolReference();
-            if (symReference->getSymbol()->isVolatile())
-               {
-               nodeHasSync = true;
-               nodeHasVolatile = true;
-               }
+            nodeHasSync = symReference->getSymbol()->isAcquireRelease();
```

Comment:
This all looks okay to me, but given that we have had subtle issues with flush elimination, I thought I would tag @klangman for his awareness, and more importantly, if he has some concern.